osu.Game/Rulesets/Mods/IApplicableToScreen.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace osu.Game.Rulesets.Mods
+{
+    /// <summary>
+    /// An interface for a mod which can temporarily override screen settings.
+    /// </summary>
+    public interface IApplicableToScreen : IApplicableMod
```

Comment:
this interface is too specific. you should be providing access to the required instances, not exposing the three things specifically required by cinema mod alone.